gen/installer/azure.py

```diff
@@ -150,8 +149,7 @@ def gen_templates(user_args, arm_template):
         results.templates[arm_template + '.json'],
         variant_cloudconfig['master'],
         variant_cloudconfig['slave'],
-        variant_cloudconfig['slave_public']
-        )
+        variant_cloudconfig['slave_public'])
```

Comment:
I prefer the first one style suggested above.. With the second style, if you have nested closing brackets, then it will end up indented each in one line.